SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+The TFTP Server Daemon project team and community take security bugs in this repository seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+
+## Reporting a Vulnerability
+To report a security issue, please use one of the following methods:
+- **GitHub Security Advisory:** Use the GitHub Security Advisory ["Report a Vulnerability"](https://github.com/altugbakan/rs-tftpd/security/advisories/new) tab on our GitHub repository.
+- **Send an Email:** Send your report to mail@alt.ug. We recommend encrypting the email if possible.
```

Comment:
You can omit this tab, I think it will be best if people use the official GitHub channel.

Reply:
I meant only the security advisory, not public issues. Thanks for the clarification.